Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -706,39 +706,99 @@ string stdInClientHandle
             SendEvent(new AP.ModulesChangedEvent());
         }
 
-        /// <summary>
-        /// Creates a new ProjectEntry for the collection of buffers.
-        /// </summary>
-        internal async void ReAnalyzeTextBuffers(BufferParser oldParser) {
-            ITextBuffer[] buffers = oldParser.Buffers;
-            if (buffers.Length > 0) {
-                if (!oldParser.AnalysisEntry.SuppressErrorList) {
-                    _services.ErrorTaskProvider?.ClearErrorSource(oldParser.AnalysisEntry, ParserTaskMoniker);
-                    _services.ErrorTaskProvider?.ClearErrorSource(oldParser.AnalysisEntry, UnresolvedImportMoniker);
-                    _services.CommentTaskProvider?.ClearErrorSource(oldParser.AnalysisEntry, ParserTaskMoniker);
+        internal void ReanalyzeEntry(AnalysisEntry entry) {
```

Comment:
>ReanalyzeEntry [](start = 22, length = 14)

Is anyone calling this? And does this really reanalyze the entry?

Reply:
No, it's dead code now, replaced by the "Transfer[File]FromOldAnalyzer" functions (it used to be called when you renamed a source file).

And I also noticed the `OnModulesChanged` event handler is no longer called... need to figure out when we should trigger that.

Reply:
Heh, that's been unused for more than a few months now :)